articles/quickstart/spa/_includes/_calling_api_create_scope.md

```diff
@@ -1,7 +1,11 @@
 ## Add a Scope
 
-When an `access_token` is received after a user authenticates in your app, it is simply an indication that they have proven their identity. By default, however, the `access_token` does not contain any **authorization** information. To properly limit access to your resources based on authorization, you must use [scopes](https://auth0.com/docs/api-auth/tutorials/adoption/api-tokens).
+By default, the access token does not contain any authorization information. To limit access to your resources based on authorization, you must use scopes. Read more about scopes in the  [api tokens documentation](https://auth0.com/docs/api-auth/tutorials/adoption/api-tokens).
 
-In the settings area for your API within the Auth0 dashboard, navigate to the **Scopes** tab. Add any scopes necessary to limit access to your API resources. While the naming of the scopes is at your discretion, a common pattern is to use `<action>:<resource>`. The example here will use a `read:messages` scope.
+In the Auth0 dashboard, in the **APIs** section, click **Scopes**. Add any scopes you need to limit access to your API resources.
+
+::: note
+You can give any names to your scopes. `<action>:<resource>` is a common pattern. The example below uses the name `read:messages` for a scope.
+:::
 
 ![create scope](/media/articles/api-auth/create-scope.png)
```

Comment:
@aaguiarz the screenshot shows the same note that is in the content. I rephrased the note. Would it be possible to change the content of the note that appears in the interface as well? The revised note says:
By default, any user can ask for any scope you defined. You can implement access policies to limit this behavior using Rules(link).